modules/tabs-v2/widgets/tabs-v2.php

```diff
@@ -405,59 +407,282 @@ protected function register_controls() {
 			'tab' => Controls_Manager::TAB_STYLE,
 		] );
 
-		$this->add_control( 'border_width', [
-			'label' => esc_html__( 'Border Width', 'elementor' ),
+		$this->add_responsive_control( 'tabs_title_spacing', [
+			'label' => esc_html__( 'Spacing', 'elementor' ),
 			'type' => Controls_Manager::SLIDER,
+			'range' => [
+				'px' => [
+					'min' => 0,
+					'max' => 400,
+				],
+			],
 			'default' => [
-				'size' => 1,
+				'unit' => 'px',
+			],
+			'size_units' => [ 'px' ],
+			'selectors' => [
+				'{{WRAPPER}}' => '--tabs-v2-gap: {{SIZE}}{{UNIT}}',
 			],
+		] );
+
+		$this->add_responsive_control( 'tabs_title_space_between', [
+			'label' => esc_html__( 'Space Between', 'elementor' ),
+			'type' => Controls_Manager::SLIDER,
 			'range' => [
 				'px' => [
 					'min' => 0,
-					'max' => 10,
+					'max' => 400,
 				],
 			],
+			'default' => [
+				'unit' => 'px',
+			],
+			'size_units' => [ 'px' ],
 			'selectors' => [
-				'{{WRAPPER}}' => '--tabs-v2-border-width: {{SIZE}}{{UNIT}};',
+				'{{WRAPPER}}' => '--tabs-v2-title-gap: {{SIZE}}{{UNIT}}',
 			],
 		] );
 
-		$this->add_control( 'border_color', [
-			'label' => esc_html__( 'Border Color', 'elementor' ),
+		$this->add_control( 'tabs_title_border_type', [
```

Comment:
For this widget we need to split controls for 'normal', 'focus' and 'active' state. Therefore I couldn't use the group control.

Reply:
Can't you just use 3 Border Group Controls, one for each state?

Reply:
![image](https://user-images.githubusercontent.com/94352322/176647247-5b57330f-ff53-411c-be35-f72a5b51677d.png)

We can't use the border group controls, because in the specs the Border Style is seperated from the other border controls settings. 

This doesn't match the structure of the border group controls.